python/orca/dev/test/run-pytests-ray.sh

```diff
@@ -49,23 +49,31 @@ python -m pytest -v test/bigdl/orca/learn/ray \
 exit_status_2=$?
 if [ $exit_status_2 -ne 0 ];
 then
-   exit $exit_status_2
+    exit $exit_status_2
 fi
 
 echo "Running orca learn tf2 ray backend tests"
 python -m pytest -v test/bigdl/orca/learn/ray/tf/test_tf2estimator_ray_backend.py
 exit_status_3=$?
 if [ $exit_status_3 -ne 0 ];
 then
-   exit $exit_status_3
+    exit $exit_status_3
 fi
 
 echo "Running orca data ray related tests"
 python -m pytest -v test/bigdl/orca/data/ray
 exit_status_4=$?
 if [ $exit_status_4 -ne 0 ];
 then
-exit $exit_status_4
+    exit $exit_status_4
 fi
 
 ray stop -f
+
+echo "Running orca learn tf2 spark backend tests"
+python -m pytest -v test/bigdl/orca/learn/spark/test_tf_spark_estimator.py
+exit_status_5=$?
+if [ $exit_status_5 -ne 0 ];
```

Comment:
Actually `if [ $exit_status_5 -ne 0 ];` isn't necessary? `set -ex` makes the scripts exit automatically on some errors. But it's beautiful to keep consistency! 😆